src/swarm-tui/Swarm/TUI/Controller.hs

```diff
@@ -350,6 +350,19 @@ handleMainEvent forceRedraw ev = do
           when shouldUpdateCursor $
             uiGameplay . uiWorldCursor .= mouseCoordsM
         REPLInput -> handleREPLEvent ev
+        (UIShortcut "Help") -> playStateWithMenu $ toggleModal HelpModal
+        (UIShortcut "Robots") -> playStateWithMenu $ toggleModal RobotsModal
+        (UIShortcut "Commands") -> playStateWithMenu $ toggleDiscoveryNotificationModal CommandsModal availableCommands
```

Comment:
Is there a reason that the Recipes (F3) and Messages (F5) shortcuts are not included as clickable?

Reply:
Whoops.  I'll add those asap.